src/command_modules/azure-cli-acr/README.rst

```diff
@@ -0,0 +1,205 @@
+Microsoft Azure CLI 'acr' Command Module
+==================================
+
+This package has [not] been tested [much] with Python 2.7, 3.4 and 3.5.
+
+Commands to manage Azure container registries
```

Comment:
I think we can still have the readme, but not have full command reference in it.   User would run the command help if required.